src/status_im2/contexts/communities/home/style.cljs

```diff
@@ -42,3 +43,34 @@
    :left        0
    :padding-top top})
 
+;;;; CARD ANIMATION
+(def card-bottom-override {:margin-bottom 16}) ; Original 8 + 8 from tabs top padding
+(def card-height (+ 56 16)) ; Card height + its vertical margins
+(def card-total-height (+ card-height 8)) ; added 8 from tabs top padding
+(def card-opacity-factor (/ 100 card-height 100))
+
+(defn- value-in-range
+  "Returns `num` if is in the range [`lower-bound` `upper-bound`]
+  if `num` exceeds a given bound, then returns the bound exceeded."
+  [num [lower-bound upper-bound]]
+  (max lower-bound (min num upper-bound)))
+
+(defn set-animated-card-values
+  [{:keys [scroll-offset height translation-y opacity]}]
+  (let [new-height        (- card-total-height scroll-offset)
+        new-opacity       (* (- card-height scroll-offset) card-opacity-factor)
+        new-translation-y (- scroll-offset)]
+    (reanimated/set-shared-value height (value-in-range new-height [0 card-total-height]))
+    (reanimated/set-shared-value opacity (value-in-range new-opacity [0 1]))
+    (reanimated/set-shared-value translation-y (value-in-range new-translation-y
+                                                               [(- card-total-height) 0]))))
```

Comment:
`set-animated-card-values` is called in the `on-scroll` callback to update the animation of the header.